packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -0,0 +1,146 @@
+import Colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Dialog from 'src/components/Dialog'
+import { Namespaces } from 'src/i18n'
+import InfoIcon from 'src/icons/InfoIcon'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { tokensWithBalanceSelector, totalTokenBalanceSelector } from 'src/tokens/selectors'
+
+function TokenBalance() {
+  const tokenBalances = useSelector(tokensWithBalanceSelector)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const totalBalance = useSelector(totalTokenBalanceSelector)
+  if (tokenBalances.length == 0) {
+    return (
+      <Text style={styles.balance}>
+        {localCurrencySymbol}
+        {'0.00'}
+      </Text>
+    )
+  } else if (tokenBalances.length == 1) {
+    return (
+      <View style={styles.oneBalance}>
+        <Image source={{ uri: tokenBalances[0].imageUrl }} style={styles.tokenImg} />
+        <View style={styles.column}>
+          <Text style={styles.balance}>
+            {localCurrencySymbol}
+            {totalBalance}
+          </Text>
+          <Text style={styles.tokenBalance}>
+            {new BigNumber(tokenBalances[0].balance).toFixed(2)} {tokenBalances[0].name}
```

Comment:
For some tokens just two decimals son't be enough :/ For example, for BTC $10 are `0,00016 BTC`. There's a `calculateDecimalsToShow` function in `TokenDisplay` you can use to figure out how many decimals to show.